tyk-hybrid/templates/secret-certs.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.secrets.useSecretName }}
```

Comment:
Here it needs to ".Values.secrets.useCertSecretName" instead of ".Values.secrets.useSecretName" so that default cert will not be created if we pass an external k8's secret name.